lib/account.js

```diff
@@ -0,0 +1,32 @@
+var account = require('./locales/common/account');
```

Comment:
Why are we not using the `lib/datasets` folder here? Why are we requiring this directly in the controller?

Reply:
To avoid repetition. I figured that since the accounts are not dependent on the locale set.
Then there might not be need for repeating in all locales.

What do you think @andela-rbabalola ?

Reply:
Oh ok @BolajiOlajide that's fine. I'll approve the PR